### PR TITLE
[OCSI]: Adds new author ruilin808

### PR DESCRIPTION
**Pull request by ruilin808**
### Description
Adds new author ruilin808
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by cwperks — APPROVED** [review]
LGTM!

File: _authors/osci_2023/ruilin808.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Ruizhi (Ray) Lin
+short_name: ruilin808
+twitter: NA
```
Comment (dtaivpp) [inline _authors/osci_2023/ruilin808.markdown:4]: Hey can you remove this line and we will merge!

**Review by dtaivpp — APPROVED** [review]
Lo

**Comment by dtaivpp** [comment]
Closing as we are past deadline and there are outstanding changes.

**Pull request closed without merging**